### PR TITLE
[DNM] bluestore: Switch to new style config options

### PR DESCRIPTION
**Pull request by shinobu-x**
src/os/bluestore/BitAllocator.cc
  src/os/bluestore/BitmapFreelistManager.cc
  src/os/bluestore/KernelDevice.cc
  src/os/bluestore/StupidAllocator.cc

Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

**Comment by shinobu-x** [comment]
test this please

**Comment by shinobu-x** [comment]
test this please

File: src/os/bluestore/KernelDevice.cc [hunk 1]
```diff
@@ -631,8 +645,8 @@ int KernelDevice::aio_write(
     ioc->pending_aios.push_back(aio_t(ioc, fd_direct));
     ++ioc->num_pending;
     aio_t& aio = ioc->pending_aios.back();
-    if (cct->_conf->bdev_inject_crash &&
-	rand() % cct->_conf->bdev_inject_crash == 0) {
+    auto bdev_inject_crash = cct->_conf->get_val<int64_t>("bdev_inject_crash");
+    if (bdev_inject_crash && rand() % bdev_inject_crash == 0) {
```
Comment (amitkumar50) [inline src/os/bluestore/KernelDevice.cc]: NIT:tab

**Comment by liewegas** [comment]
We don't want ot do wholesale conversions to get_val since it is slower

**Pull request closed without merging**